content/05.prior-knowledge.md

```diff
@@ -1,34 +1,33 @@
 
 In the academic rare disease space, there a few pioneering examples of ML-based mining of knowledge graphs to repurpose drugs[@doi:10.1101/727925] and classify rare diseases[@doi:10.1186/s12911-019-0938-1].
-These studies make it clear that there are some challenges in using machine learning using graph databases in rare disease.
-For example, these papers rely on a gold standard dataset to validate the performance of the models; often, there are not robust gold standard datasets available for individual rare diseases.
+These studies make it clear that there are challenges in using machine learning using graph databases in rare disease.
+For example, these projects rely on a gold standard dataset to validate the performance of the models; often, there are not robust gold standard datasets available for individual rare diseases.
 They also evaluate rare diseases in an unbiased manner, rather than interrogating a specific disease of interest.
 Consequently, it is not yet clear how effective these approaches, and knowledge graphs in general, are in studying a specific disease of interest; more work needs to be done to identify methods that can provide actionable insights for a specific rare disease application. 
 
-Beyond the aforementioned studies, there are few examples of studies in the public domain that leverage knowledge graphs to characterize rare disease. 
-Private entities (e.g. healx, Boehringer Ingelheim, DrugBankPlus) are performing an undisclosed amount of work to create proprietary rare disease knowledge graphs for ML-based drug discovery applications.  
-The existence of private companies pursuing this idea, as well as the availability of several public knowledge graphs with relevance to rare disease, suggests to us that this is a likely fruitful and untapped area of rare disease research in the public sphere. 
+Beyond the aforementioned studies, there are not many examples of studies in the public domain that leverage knowledge graphs to characterize rare diseases. 
+Private entities (e.g. healx, Boehringer Ingelheim), however, are performing an undisclosed amount of work to create proprietary rare disease knowledge graphs for ML-based drug discovery applications.
+The existence of private companies pursuing this idea, as well as the availability of public biomedical knowledge graphs, suggests that this is likely a fruitful untapped area of rare disease research in the public arena. 
 More work needs to be done to assess 1) which graphs and graph features capture the salient information about rare diseases, 2) the utility of ML methods to obtain actionable insights about rare diseases and 3) which problems - like drug discovery, identification of novel rare diseases, or assessment of genotype-phenotype relationships - can be interrogated using ML of knowledge graphs.
 
 
 #### Wisdom of the crowd: rare disease applications of ensemble methods
```

Comment:
I think this part is better suited for the transition from model-complexity section to prior-knowledge section

Reply:
This isn't clear to me - do you mean we should talk about ensemble models before any of the individual methods?

Reply:
My feeling is that the bit about ensembling is relating more closely to the present model-complexity section as a technique for stable feature selection, compared to prior-knowledge section.

Reply:
similarly for the binary vs non-binary classification

Reply:
hmm, interesting. I'll file it as an issue!